sanic/server.py

```diff
@@ -154,6 +157,8 @@ def data_received(self, data):
             self.headers = []
             self.parser = HttpRequestParser(self)
 
+        self.state['requests_count'] = self.state['requests_count'] + 1
```

Comment:
How about on_message_complete ?

I can update to connection_made if you guys all agree on this.